benchmarks/Ruby/harness-rbx.rb

```diff
@@ -55,7 +55,7 @@ def load_benchmark_suite(benchmark_name)
       benchmark_file = benchmark_name.gsub(/([a-z])([A-Z])/) { "#{$1}-#{$2}" }.downcase
     end
     unless require_relative(benchmark_file)
-      raise "failed loading #{benchmark_file}"
```

Comment:
Hm, is this really better? What happens if you name a benchmark that doesn't exist?

Reply:
`require_relative` raises a LoadError in that case (as the commit message says)

Reply:
Here are the docs just in case:
```
=== Implementation from Kernel
------------------------------------------------------------------------
  require_relative(string) -> true or false

------------------------------------------------------------------------

Ruby tries to load the library named string relative to the
requiring file's path.  If the file's path cannot be determined a
LoadError is raised. If a file is loaded true is returned and false
otherwise.
```
They are pretty bad :/ and I understand your confusion now.
`require` docs are a bit clearer:
```
=== Implementation from Kernel
------------------------------------------------------------------------
  require(name)    -> true or false

------------------------------------------------------------------------

Loads the given name, returning true if successful and false if the
feature is already loaded.
...
 If the file named cannot be found, a
LoadError will be raised.
```

Reply:
https://github.com/ruby/ruby/commit/1f067e775d8629ac3b33f96dd885d5f996e787f3